### ISSUE TITLE
There might be a conflict between 'shippingType' in PaymentOptions and WebIDL binding.

### ISSUE THREAD

**Issue by romandev** [w3c/browser-payment-api#271]
In the current spec, 'shippingType' in PaymentOptions is defined as enum PaymentShippingType.

BTW, According to PaymentRequest constructor algorithm 22[1], if options.shippingType is not a valid PaymentShippingType value then set the shippingType attribute on request to 'shipping'.

On the other hand, according to webidl spec[2], it causes a TypeError if passing a invalid enum value as operation argument. It means that the constructor can stop due to a exception that occured in binding side.

In other words, the former is conflicting with the latter. (But I'm not sure..)
IMHO, if the shippingType changes into DOMString, this issue might be resolved.

What do you think?

[1] https://w3c.github.io/browser-payment-api/#h-note5
[2] http://heycam.github.io/webidl/#example-363ff014

**Comment by adrianba** [comment]
Good catch - will need to update this.
